modules/kmm-creating-module-cr.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-creating-module-cr_{context}"]
+
+= The Module Custom Resource Definition
```

Comment:
@StephenJamesSmith, we're using lowercase for CRD: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource-definition-crd

Reply:
Made this change.